bsdploy/bootstrap-files/pkg.conf

```diff
@@ -1 +1,8 @@
 ASSUME_ALWAYS_YES: YES
+REPO_AUTOUPDATE: NO
```

Comment:
This change isn't documented. Either it should also be an option or go inside the ``if`` and be documented with the proxy support.